package-lock.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@heroku/socksv5",
```

Comment:
This file probably needs to be re-generated?

Reply:
@himanshusinghs Correct, nice catch! Updated that commit in 66f45dddb9f8860eb9040568076fe3fc40ca8e5c